tests/components/matter/test_number.py

```diff
@@ -0,0 +1,56 @@
+"""Test Matter number entities."""
+
+from unittest.mock import MagicMock
+
+from matter_server.client.models.node import MatterNode
+import pytest
+
+from homeassistant.core import HomeAssistant
+
+from .common import (
+    set_node_attribute,
+    setup_integration_with_node_fixture,
+    trigger_subscription_callback,
+)
+
+
+@pytest.fixture(name="light_node")
+async def dimmable_light_node_fixture(
+    hass: HomeAssistant, matter_client: MagicMock
+) -> MatterNode:
+    """Fixture for a flow sensor node."""
+    return await setup_integration_with_node_fixture(
+        hass, "dimmable-light", matter_client
+    )
+
+
+# This tests needs to be adjusted to remove lingering tasks
```

Comment:
This seems like a todo?

Reply:
Its a comment on all our tests, maybe we should just not replicate it everywhere

Reply:
Ok lets ignore for now.

Reply:
Lets clean it up at a later stage (all at once) if we have confirmed with Martin what's needed